jobs/views.py

```diff
@@ -64,9 +66,44 @@ def form_valid(self, form):
         return super(JobCreate, self).form_valid(form)
 
 
-class JobList(ListView, JobActiveMixin, FilterableList):
+class JobList(ListView, JobActiveMixin, FilterQuerySetMixin, FilterableList):
     model = Job
     paginate_by = 20
+    filter_fields = {
+        'title': 'title__icontains',
+        'location': 'location__icontains',
+        'seniority': 'seniority',
+        'remote_work': 'remote_work',
+        'company': 'company'
+    }
+
+    def get_context_data(self, **kwargs):
+        context = super(JobList, self).get_context_data(**kwargs)
+        context['search_jobs_form'] = JobSearchForm(self.request.GET or None)
+        return context
+
+    def get_queryset(self):
```

Comment:
1. Esta función no está testeada.
2. ¿Por qué hace `if pattern in self.request.GET['created']:` en vez de usar `==`? La interfaz no es clara, soporta cosas como 'foo-created-bar'.
3. Los posibles valores de `'created'` están hardcoded cuando en realidad corresponden a los `JobSearchForm.CREATED_CHOICES`

Reply:
@cmdelatorre estamos laburando en esto :+1:  gracias por el review! :)

Reply:
@cmdelatorre lo del `if created in self.request.GET` no lo veo mal, se fija si estan pasando en el GET un param para filtrar por `created`. En que le estamos pifiando? Respecto de usar los choices del JobSearchForm ahi lo ajustamos :+1: .

Reply:
Ya entiendo lo de `self.request.GET`, mala mia.